polaris.shopify.com/src/components/Subnav/Subnav.tsx

```diff
@@ -91,4 +98,32 @@ function getObjectValue<T>(obj: any, path: string): T | undefined {
   return value;
 }
 
+const PolarisIcon = () => (
```

Comment:
We will need this PolarisIcon and the HexagonIcon for other parts of the site as well. Can we move them to shared location?

Reply:
I didn't see one either, but `src/icons` is a good starting point 💯